hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -418,4 +436,37 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(volumeName, bucketName, keyName);
   }
+
+  /**
+   * Return a new copy of the object.
+   */
+  public OmKeyInfo copyObject() {
```

Comment:
Why not implement `Cloneable`?

Reply:
Yes, it can be. As for Cloneable if it is not final class, I need to call super.clone() due to Cloneable semantics. And also to have the same way of this for all entities.